.github/workflows/remove-from-stage.yml

```diff
@@ -0,0 +1,25 @@
+name: Remove from Staging
+
+on: 
+  delete:
+    branches:
+      - '*'
+      - '!main'
+
+jobs:
+  remove:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Extract branch name
+      shell: bash
+      run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      id: extract_branch
+    - name: Remove from s3
+      uses: vitorsgomes/s3-rm-action@master
+      with:
+        args: --recursive
+      env:
+        AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        PATH_TO_DELETE: 'trackerblocking/config/staging/${{ steps.extract_branch.outputs.branch }}'
```

Comment:
Is this a prefix match or path? I wonder if there exists a branch `sam` and `samv2` and I delete `sam` if this action might delete the files in `samv2` too. I think a trailing slash would resolve it. You don't have to delete the directory on s3 as it simply ceases to exist once all the files in it are deleted.

Reply:
I believe its a path but I'll add the trailing slash to eliminate ambiguity.